README.md

```diff
@@ -46,4 +46,4 @@ The below image is an example of the resulting html, which is called on an recur
 
 The "cpu_usage" on the top indicates the corresponding cpu usage at each time period. The "memory_usage" have two colors, representing rss(Resident Set Size, the non-swapped physical memory) and vms(Virtual Memory Size). You can see the details by clicking on one of the periods.
 
-[![example_img](https://github.com/in-the-ocean/vizplugins/blob/readme/img/fib.png)](https://github.com/in-the-ocean/vizplugins/blob/readme/img/fib.png)
\ No newline at end of file
+[![example_img](https://github.com/in-the-ocean/vizplugins/blob/readme/img/fib.png)](https://github.com/in-the-ocean/vizplugins/blob/readme/img/fib.png)
```

Comment:
What did you do on this line?

Reply:
Sorry for the confusion. I think it is because `newline at end of file`.
I used github `edit this file` to in browser, and it seems automatically created a line. I reverted it by a new commit.

commit b6c5df314e2d6349a8ff4a1619d635151fdcf67c, it end with $

![image](https://github.com/gaogaotiantian/vizplugins/assets/19520698/40cd3a4f-8c92-48ad-a51a-5d59b0b3c5b5)